transfers/models.py

```diff
@@ -22,6 +24,10 @@ def __repr__(self):
 
 
 def init(databasefile):
+    if not isfile(databasefile):
+        # We create the file
+        with open(databasefile, "a"):
+            pass
```

Comment:
Here, the other solution is just to put
```python
with open(databasefile, "a"):
    pass
```
(so remove the condition)

But the disadvantage is that it will open the file everytime (but the code is simpler).

There is also the solution to use this function: https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch
Unfortunately it doesn't seem to exist in python 2.